recipes/clincnv/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+# Copy all source files from SRC_DIR into the bin folder
+mkdir -p "$PREFIX/bin/clincnv/"
+cp -r "$SRC_DIR"/* "$PREFIX/bin/clincnv/"
```

Comment:
I'm also a bit concerned about this. The tarball contains many things that seem superfluous to running the scripts.

If you want to copy everything like this, then the upstream repository should be cleaned up to remove unnecessary components from the tarball. For example, maybe create a new branch that only includes required code.

Alternatively, this should be changed to only copy the minimum necessary files.

Reply:
Hi @mfansler I just edited the yaml file to update which files are copied over to the necessary ones. Would you mind taking another look so this can be merged in?